cmd/example/register-cluster_complicated-exemple.json

```diff
@@ -0,0 +1,280 @@
+          },
+          {
+            "key": "fluent-bit.config.extra",
+            "value": "\n[INPUT]\n    Name              tail\n    Tag               dex.*\n    Path              /var/log/containers/*_dex-*.log\n    DB                /var/log/flb_kube_dex.db\n    parser            docker\n    Mem_Buf_Limit     5MB\n    Skip_Long_Lines   On\n    Refresh_Interval  10\n[FILTER]\n    Name    lua\n    Match   dex.*\n    script  script.lua\n    call    reformat\n[FILTER]\n    Name    grep\n    Match   dex.*\n    Regex   time .*\n[FILTER]\n    Name    grep\n    Match   dex.*\n    Regex   data .*\\\"xsuaa\n[OUTPUT]\n    Name             http\n    Match            dex.*\n    Retry_Limit      False\n    Host             api.auditlog.cf.stagingaws.hanavlab.ondemand.com\n    Port             8081\n    URI              /audit-log/v2/security-events\n    Header           Content-Type application/json\n    HTTP_User        ${AUDITLOG_USER}\n    HTTP_Passwd      ${AUDITLOG_PASSWD}\n    Format           json_stream\n    tls              on\n"
+          },
+          {
+            "key": "fluent-bit.config.secrets.AUDITLOG_USER",
+            "value": "10cdf4a38423fc1d"
+          },
+          {
+            "key": "fluent-bit.config.secrets.AUDITLOG_PASSWD",
+            "value": "fcf70646fed58151"
+          },
+          {
+            "key": "fluent-bit.externalServiceEntry.resolution",
+            "value": "DNS"
+          },
+          {
+            "key": "fluent-bit.externalServiceEntry.hosts",
+            "value": "- api.auditlog.cf.stagingaws.hanavlab.ondemand.com"
+          },
+          {
+            "key": "fluent-bit.externalServiceEntry.ports",
+            "value": "- number: 8081\n  name: https\n  protocol: TLS"
+          }
+        ]
+      }
+    ],
+    "administrators": [
+      "test2@test.pl"
+    ]
+  },
+  "metdata": {
+    "runtimeID": "6f8ec1bc-f300-4f00-bb65-6fec5dcd0ba5",
+    "globalAccountID": "3e64ebae-38b5-46a0-b1ed-9ccee153a0ae",
+    "subAccountID": "f7d129fa-b2fe-11eb-8529-0242ac130003",
+    "serviceID": "47c9dcbf-ff30-448e-ab36-d3bad66ba281",
+    "servicePlanID": "4deee563-e5ec-4731-b9b1-53b42d855f0c",
+    "shootName": "4deee563"
```

Comment:
I would add instanceID. Usually runtimeID is enough - unique RuntimeID -> unique instance ID. The relation is 1 to 1. Biut in the scenario with suspension one instance could have more than one runtimes (one runtime at the same time, but many runtimes in the history). Such instance ID could makes some operations easier in the future

Reply:
Done